src/api-service/__app__/onefuzzlib/repro.py

```diff
@@ -71,10 +75,12 @@ def get_vm(self) -> VM:
             sku=vm_config.sku,
             image=vm_config.image,
             auth=self.auth,
+            tags=tags,
         )
 
     def init(self) -> None:
-        vm = self.get_vm()
+        config = InstanceConfig.fetch()
```

Comment:
Since you only use the config within self.get_vm(), why not make the call there?

Reply:
oh, nevermind, I see it later on.